Std/Tactic/Omega/OmegaM.lean

```diff
@@ -129,6 +129,15 @@ def analyzeAtom (e : Expr) : OmegaM (HashSet Expr) := do
   | (``Nat.cast, #[.const ``Int [], _, e']) =>
     -- Casts of natural numbers are non-negative.
     let mut r := {Expr.app (.const ``Int.ofNat_nonneg []) e'}
+    match e'.getAppFnArgs with
+    | (``HPow.hPow, #[_, _, _, _, b, exp]) =>
+      match b == toExpr (2 : Nat), exp.getAppFnArgs with
+      | true, (``Nat.succ, #[n]) =>
+        -- 2^(Nat.succ n) = 2^n + 2^n
```

Comment:
Here, too, we should probably match on `_ + 1`, or both `succ _` and `_ + 1`. Or, does omega have some normalization in place?

Reply:
Sure, let me change the normalization to deal with both cases.

Reply:
I've changed the normalization, but this is probably superceded by the other suggestion.